### ISSUE TITLE
Improve Travis CI integration

### ISSUE THREAD

**Issue by igorsantos07** [laravelbook/ardent#268]
It's imperative we include Travis CI support to verify code styling issues, as PRs usually come with different code styles and that reduces code quality.

In the future we can use this integration for tests as well.

Suggestion: verify how Faker does it.

**Comment by GrahamCampbell** [comment]
> It's imperative we include Travis CI support to verify code styling issues

Check out https://styleci.io/. :)

**Comment by GrahamCampbell** [comment]
That way you can keep travis reserved for the unit tests, and have styleci giving an additional green/red light on PRs.

**Comment by igorsantos07** [comment]
That's awesome! I didn't know that service.
I though on Travis as it had integration and I saw it working fine on Faker.

**Comment by igorsantos07** [comment]
Unable to implement StyleCI. I'm not the owner of the project... And I'm quite sure @laravelbook won't even try to reply us.

**Comment by igorsantos07** [comment]
I'll send him an email asking for help. TravisCI behaves the same way (why did I though it would be different? lol)

**Comment by GrahamCampbell** [comment]
@igorsantos07 If you need any help, feel free to contact me, and I can help you get set up, though I can't manually enable this repo on StyleCI for you until someone with admin permissions on this repo has an account on StyleCI because we'd be using their authentication token in order to use GitHub's status API. Non-admin users don't have permission to write to the status API for their repos you see. :)

**Comment by igorsantos07** [comment]
Yeah... That's way too bad. Not because of the API, but the other way
around: the original owner of this code vanished. I couldn't find any
reference about him on the Web.
I don't think I have any access to packagist stuff as well, so it's also
hard to transfer the composer package to my fork.

I'll try to investigate things later today and see what I can came up with
:(
